release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2419 @@
+----
+
+[[ocp-36-registry-rest-endpoint-for-reading-writing-image-signatures]]
+==== Registry REST Endpoint for Reading and Writing Image Signatures
+
+There is now a programmable way to read and write signatures using only the docker registry API.
+
+To read, you must be authenticated to the registry.
+
+----
+PUT /extensions/v2/{namespace}/{name}/signatures/{digest}
+$ curl http://<user>:<token>@<registry-endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
+
+JSON:
+{
+  "version": 2,
+  "type":    "atomic",
+  "name":    "sha256:4028782c08eae4a8c9a28bf661c0a8d1c2fc8e19dbaae2b018b21011197e1484@cddeb7006d914716e2728000746a0b23",
+  "content": "<base64 encoded signature>",
+}
+----
+
+To write, you must have the `image-signer` role.
+
+----
+GET /extensions/v2/{namespace}/{name}/signatures/{digest}
+$ curl http://<user>:<token>@<registry-endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
+
+
+{
+  "signatures": [
+  {
+    "version": 2,
+    "type":    "atomic",
+    "name":    "sha256:4028782c08eae4a8c9a28bf661c0a8d1c2fc8e19dbaae2b018b21011197e1484@cddeb7006d914716e2728000746a0b23",
+    "content": "<base64 encoded signature>",
+  }
+  ]
+}
+----
+
+[[ocp-36-platform-management]]
+=== Platform Management
+
+[[ocp-36-require-explicit-quota-to-consume-a-resource]]
+==== Require Explicit Quota to Consume a Resource (Technology Preview)
+
+If a resource is not managed by quota, a user has no restriction on the amount
+of resource that can be consumed. For example, if there is no quota on storage
+related to the gold storage class, the amount of gold storage a project can
+create is unbounded.
+
+See xref:../admin_guide/quota.adoc#limited-resources-quota[Setting Quotas] for
+more information.
+
+[[ocp-36-storage]]
+=== Storage
+
+[[ocp-36-aws-efs-provisioner]]
+==== AWS EFS Provisioner
+
+The AWS EFS Provisioner allows you to dynamically use the AWS EFS endpoint to
+get NFS remote persistent volumes on AWS.
+
+It leverages the
+link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#provisioner[external
+dynamic provisioner interface]. It is provided as a `docker` image that you
+configure with a `configMap` and deploy on {product-title}. Then, you can use a
+storage class with the appropriate configuration.
+
+.Storage Class Example
+----
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: slow
+provisioner: foobar.io/aws-efs
+parameters:
+  gidMin: "40000"
+  gidMax: "50000"
+----
+`gidMin` and `gidMax` are the minimum and maximum values, respectively, of the
+GID range for the storage class. A unique value (GID) in this range (`gidMin` to
+`gidMax`) is used for dynamically provisioned volumes.
+
+[[ocp-36-vmware-vsphere-storage]]
+==== VMware vSphere Storage
+
+VMware vSphere Storage allows you to dynamically use the VMware vSphere storage
+options ranging from VSANDatastore, ext3, vmdk, and VSAN while honoring vSphere
```

Comment:
@childsb @gnufied Can you please review? I'm not sure whether we have decided to announce vSphere support officially.